src/views/Space/Overview.vue

```diff
@@ -1,19 +1,37 @@
 <script setup lang="ts">
 import { onMounted, computed } from 'vue';
 import { useProposalsStore } from '@/stores/proposals';
-import { Space } from '@/types';
+import { Space, Proposal as ProposalType } from '@/types';
+
+const PROPOSALS_LIMIT = 4;
```

Comment:
We need to fetch one more than we want to display to be able to detect if we need See more link.